Gruntfile.js

```diff
@@ -89,6 +89,15 @@ module.exports = function(grunt) {
       angular1_4: {
         configFile: 'config/karma.conf.angular.1.4.js'
       }
+    },
+    coveralls: {
+      options: {
+        debug: true,
+        coverageDir: 'coverage',
+        dryRun: true,
```

Comment:
>Using this option you can run coverage without sending data to the coveralls service and instead writing the results to coveralls.json.

Should it be false? because we need to send data to coveralls service.